scripts/vendor

```diff
@@ -18,12 +18,12 @@ init() {
   cat > go.mod <<EOL
 module github.com/docker/cli
 
-go 1.18
+go 1.19
 EOL
 }
 
 update() {
-  (set -x ; go mod tidy -compat=1.18 -modfile=vendor.mod; go mod vendor -modfile=vendor.mod)
+  (set -x ; go mod tidy -compat=1.19 -modfile=vendor.mod; go mod vendor -modfile=vendor.mod)
```

Comment:
honestly wondering if we still need this one (I recall it was more important between some older versions that didn't include the `// indirect` block), but I guess it does not harm

Reply:
Doesn't seem to change anything if I remove it - want me to commit that?

Reply:
Nah, I think it's fine to keep it for now, and be on the safe side in case they decide to change the format in an incompatible way.